Src/Microsoft.Dynamic/Runtime/CompilerContext.cs

```diff
@@ -22,46 +22,13 @@ namespace Microsoft.Scripting.Runtime {
     /// from this class to provide additional contextual information.
     /// </summary>
     public sealed class CompilerContext {
+        public SourceUnit SourceUnit { get; }
 
-        /// <summary>
-        /// Source unit currently being compiled in the CompilerContext
```

Comment:
We probably should, but I expect most of the codebase doesn't so I'm not too worried about it.